### ISSUE TITLE
Don't use CC BY-NC

### ISSUE THREAD

**Issue by IBPX** [Siile/Ninslash#11]
Why NC?

> If you mark your content as NC, it cannot be included in free knowledge databases like Wikipedia, into open media archives and in Open Source projects. It is often a commercial use that allows not-for-profit initiatives which work for the public good to succeed.

You won't be able to get this put in repos in Linux distros, it won't be considered open source, and this will really hinder adoption.

If you don't want people to "rip off" your work, you could use CC BY-SA, which requires them to share under the same terms, giving back to the community.

> For private individuals, there is usually no good reason for a restriction to non-commercial use only. In many cases there is very little realistic prospect that a commercial user is willing to pay money in order to use the content. A CC license has the main goal to make the content as widely spread as possible. And in achieving this the NC module is often a hindrance.

Aside from that, there's also [_this_](https://www.techdirt.com/articles/20140326/11405526695/german-court-says-creative-commons-non-commercial-licenses-must-be-purely-personal-use.shtml) in Germany.

In short, NC does not benefit anyone. I really ask you to switch to a plain CC BY-SA license, without the noncommercial part.

**Comment by Siile** [comment]
Thanks for pointing this out! I'll change the licensing to next version.

**Comment by IBPX** [comment]
Thanks! I'd recommend doing this: Licensing the code under Zlib, which is compatible with Teeworld's original license, and using Creative Commons Attribution-ShareAlike for assets, like Teeworlds.
